x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/services/jinaai/JinaAIService.java

```diff
@@ -339,6 +341,33 @@ public static InferenceServiceConfiguration get() {
             () -> {
                 var configurationMap = new HashMap<String, SettingsConfiguration>();
 
+                configurationMap.put(
+                    JinaAIServiceSettings.MODEL_ID,
+                    new SettingsConfiguration.Builder(supportedTaskTypes).setDescription(
+                        "The name of the model to use for the inference task."
+                    )
+                        .setLabel("Model ID")
+                        .setRequired(true)
```

Comment:
Required for both text embedding and rerank: https://api.jina.ai/redoc#tag/embeddings